src/mutations/system/deleteAnnouncements.ts

```diff
@@ -15,10 +15,17 @@ const resolver: MutationToDeleteAnnouncementsResolver = async (
 
   const itemIds = ids.map((id) => fromGlobalId(id).id)
 
-  await atomService.deleteMany({
-    table,
-    whereIn: ['id', itemIds],
-  })
+  await atomService
+    .deleteMany({
+      table: 'announcement_translation',
+      whereIn: ['id', itemIds],
```

Comment:
here should be `announcement_id`? from https://github.com/thematters/matters-server/blob/develop/db/migrations/20221009151817_create_announcement_translation.js#L9

the `announcement_translation` table has id but not the same one;